docs/user-guide/leverage-cli/reference/terraform.md

```diff
@@ -108,6 +108,15 @@ Open a shell into the Terraform container in the current directory. An authentic
 
 _Note:_ When `--sso` flag is used, the `--mfa` flag status is ignored.
 
+!!! example "What if I want to run a Terraform command that is not supported by the CLI?"
+
+
+    One common error you could encounter is `"Error acquiring the state lock"`, where you might need to use `force-unlock`. You can do the following:
+
+    1 - `leverage terraform shell --sso` 
+
+    2 - Then from inside the container: `terraform force-unlock LOCK-ID`
+
```

Comment:
@fgauchat-binbash Fran the content looks great, one thing that call our attention with @angelofenoglio is that there are several extra whitespaces that could be deleted, and the use of a numbered markdown list will be also recommended to follow the standard we use to.

So we'll recommend updating the code as follows please:

```
_Note:_ When `--sso` flag is used, the `--mfa` flag status is ignored.

!!! example "What if I want to run a Terraform command that is not supported by the CLI?"
    One common error you could encounter is `"Error acquiring the state lock"`, where you might need to use `force-unlock`. You can do the following:
        
    1. `leverage terraform shell --sso`
    2. Then from inside the container: `terraform force-unlock LOCK-ID`

---
## `format`
```

CC: @diego-ojeda-binbash